walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc

```diff
@@ -0,0 +1,182 @@
+:sso-realm-url: {user-sso-url}/auth/admin/{user-sanitized-username}/console/index.html
+:sso-adapter-docs-url: https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html/securing_applications_and_services_guide/index
+
+:rhmi-sso-name: Managed Integration SSO instance
+:customer-sso-name: End user SSO instance
+
+= Protecting Applications using Red Hat Single Sign-On (SSO)
+
+This Solution Pattern demonstrates how SSO can be used to protect web applications by providing single sign-on capabilities based on popular standards such as SAML 2.0, OpenID Connect and OAuth 2.0.
+
+NOTE: You must complete the *Integrating message-oriented middleware with a RESTful API using AMQ Online* Solution Pattern prior to starting this one.
+
+Each Red Hat Managed Integration cluster includes two SSO instances:
+
+* *{rhmi-sso-name}* provides authenication to the Managed Integration cluster.
+
+* *{customer-sso-name}* can be used by developers to secure applications they deploy on the cluster, or even on other infrastructure.
+
+
+This Solution Pattern shows you how to use the *{customer-sso-name}* to enable
+authentication and authorisation for the *Order Entry System* and the *Order
+Management System* from the *Integrating message-oriented middleware with a
+RESTful API using AMQ Online* Solution Pattern.
+
+image::images/arch.png[integration, role="integr8ly-img-responsive"]
+
+[type=walkthroughResource,serviceName=openshift]
+.Red Hat OpenShift
+****
+* link:{openshift-host}/console[Console, window="_blank"]
+* link:https://help.openshift.com/[OpenShift Online Help Center, window="_blank"]
+* link:https://blog.openshift.com/[OpenShift Blog, window="_blank"]
+****
+
+[type=walkthroughResource,serviceName=3scale]
+.End user SSO instance
```

Comment:
Cannot use a variable for this name for some reason.

Reply:
CC @pwright

Reply:
@evanshortiss  https://issues.redhat.com/browse/INTLY-4322